bedrock/mozorg/middleware.py

```diff
@@ -45,3 +46,17 @@ def process_view(self, request, view, view_args, view_kwargs):
         else:
             f = super(MozorgRequestTimingMiddleware, self)
             f.process_view(request, view, view_args, view_kwargs)
+
+
+class CrossOriginResourceSharingMiddleware(object):
```

Comment:
RE: django-cors-headers. I did see that your implementation is superior in that you have capacity for multiple regexes for URL prefixes. Considering that this is so far the only URL that needs this maybe that's okay for now? We can always fork and fix the lib if we need to later. But even if we do need multiple CORS endpoiints we can probably get by with just a slightly more complex single regex.